stingray/spectroscopy.py

```diff
@@ -155,7 +170,9 @@ def ccf_error(
     return error, avg_seg_ccf
 
 
-def get_parameters(counts, dt, model):
+def get_parameters(
+    counts: npt.ArrayLike, dt: float, model: Any
```

Comment:
I think the type of counts should be `Union[npt.ArrayLike[int],npt.ArrayLike[float]`